documentation/modules/ROOT/pages/transformations/event-changes.adoc

```diff
@@ -0,0 +1,149 @@
+:page-aliases: configuration/event-changes.adoc
+[id="event-record-changes"]
+= Event Record Changes
+
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highligher: highlight.js
+
+toc::[]
+
+ifdef::community[]
+[NOTE]
+====
+This single message transformation (SMT) is supported for only the SQL database connectors.
+====
+endif::community[]
+
+A {prodname} data change event has a complex structure that provides a wealth of information.
+However, parts of the chagne data capture pipeline may require knowledge of what fields were changed or unchanged when the event occurred in the source system.
+To provide this information, {prodname} provides a event record changes single message transformation (SMT).
+
+The event changes transformation is a
+link:https://kafka.apache.org/documentation/#connect_transforms[Kafka Connect SMT].
+
+== Change event structure
+
+
+{prodname} generates data change events that have a complex structure.
+Each event consists of three parts:
+
+* Metadata, which includes but is not limited to:
+
+** The operation that made the change
+** Source information such as the names of the database and table where the change was made
+** Time stamp for when the change was made
```

Comment:
```suggestion
** The type of operation that changed the data.
** Source information, such as the names of the database and the table in which the change occurred.
** Timestamp that identifies when the change was made.
```

Reply:
I know that this same list appears elsewhere (e.g., the event-flattening SMT docs). I'm in the process of making similar changes there as well.